### PR TITLE
with properites

### PR DESCRIPTION
**Pull request by nevermore3**

### CODE REVIEW

File: src/parser/parser.yy [hunk 1]
```diff
@@ -1664,28 +1667,33 @@ fetch_sentence
...
+        s->setOver($7);
+        s->setStep($8);
+        /* s->setWhere($9); */
         $$ = s;
     }
     ;
 
+opt_with_properites
+    : %empty { $$ = false; }
+    | KW_WITH_PROPERITES { $$ = true; }
```
Comment (CPWstatic) [inline src/parser/parser.yy:1694]: KW_WITH KW_PROPERTIES

Comment (nevermore3) [reply src/parser/parser.yy]: done

**Comment by Shylock-Hg** [comment]
Add some result test?

File: src/parser/parser.yy [hunk 2]
```diff
@@ -154,7 +154,7 @@ static constexpr size_t MAX_ABS_INTEGER = 9223372036854775808ULL;
 %token KW_ORDER KW_ASC KW_LIMIT KW_OFFSET
 %token KW_DISTINCT KW_ALL KW_OF
 %token KW_BALANCE KW_LEADER
-%token KW_SHORTEST KW_PATH
+%token KW_SHORTEST KW_PATH KW_PROPERITES
```
Comment (Shylock-Hg) [inline src/parser/parser.yy:157]: I you'd better reuse the keyword `PROPS`

Comment (nevermore3) [reply src/parser/parser.yy:157]: make sense

**Review by CPWstatic — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**